src/aframe-streetmix-parsers.js

```diff
@@ -277,6 +278,19 @@ function createFoodTruckElement (variantList, positionX) {
   return foodTruckParentEl;
 }
 
+function createOutdoorDining (variantList, positionX) {
+  const outdoorDiningParentEl = document.createElement('a-entity');
+
+  const reusableObjectEl = document.createElement('a-entity');
+  const rotationY = 180;
+  reusableObjectEl.setAttribute('position', positionX + ' 0 0');
+  reusableObjectEl.setAttribute('rotation', '0 ' + rotationY + ' 0');
```

Comment:
i think we can remove this line and 285 since we don't need to change rotation in any case?

Reply:
Sounds good :+1: